cli/bench/types.go

```diff
@@ -0,0 +1,27 @@
+package bench
+
+import (
+	bctx "context"
+	"sync"
+
+	"github.com/FZambia/tarantool"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+type requestLoopType func(ctx context.BenchCtx, tarantoolConnection *tarantool.Connection, results *Results, waitGroup *sync.WaitGroup, backgroungCtx bctx.Context)
```

Comment:
It is not ok when the line [doesn't fit on the screen](https://sun9-46.userapi.com/impg/tZX30gF9A8pj1Ur8WV7mMIWp0ysOuBInkPbJ3w/Ed5n-yFZ4bw.jpg?size=1192x78&quality=96&sign=c96b1a11267d7dee87a047e28054e29e&type=album).